guacamole/src/main/frontend/src/app/client/directives/guacClient.js

```diff
@@ -220,6 +221,9 @@ angular.module('client').directive('guacClient', [function guacClient() {
             display.showCursor(!localCursor);
             client.sendMouseState(event.state, true);
 
+            // Broadcast the mouse event
+            $rootScope.$broadcast('guacClientMouseEvent', event);
```

Comment:
I think these events should be renamed in line with other established events:
* Omit the redundant "Event" from the name of the event.
* Split into the types of events recognized by the objects firing them (`guacClientMouseDown`, `guacClientMouseUp`, etc.) so that the handler of these events can have that context without needing to implement their own state tracking, similar to the established `guacKeyDown` and `guacKeyUp`.

Reply:
How does this look?

Reply:
I like it.